tailcfg/tailcfg.go

```diff
@@ -1058,6 +1058,11 @@ type PortRange struct {
 	Last  uint16
 }
 
+// Contains returns whether port is in pr.
```

Comment:
Reports

Reply:
not applicable here, not a bool return.

Reply:
Yeah it is

Reply:
oops, i thought this was in a different place. my bad

Reply:
fixed